tests/integration.rs

```diff
@@ -8240,6 +8245,7 @@ mod common_sequential {
     }
 
     #[test]
+    #[cfg(not(feature = "mshv"))] // See issue #7437
     #[cfg_attr(target_arch = "aarch64", ignore = "See #6970")]
```

Comment:
It might be better to write the cfg attributes similar to this line. But idk if that is a big deal.